pallets/communities/src/types.rs

```diff
@@ -0,0 +1,60 @@
+use codec::MaxEncodedLen;
+use frame_support::pallet_prelude::{Decode, Encode};
+use frame_support::traits::fungibles::Inspect;
+use frame_support::{sp_runtime::BoundedVec, traits::ConstU32};
+use scale_info::TypeInfo;
+
+use crate::Config;
+use frame_system::Config as SystemConfig;
+
+pub type AccountIdOf<T> = <T as SystemConfig>::AccountId;
+pub type CommunityIdOf<T> = <T as Config>::CommunityId;
+pub type MemberRankOf<T> = <T as Config>::MemberRank;
+pub type AssetIdOf<T> = <<T as Config>::Assets as Inspect<AccountIdOf<T>>>::AssetId;
+pub type MemberListOf<T> = Vec<AccountIdOf<T>>;
```

Comment:
is this used anywhere?

Reply:
Not now. Can be used in the future, though, for things like voting lists, or if needed in a function that retrieves the members' list.

Initially, the approach was to store the member's list as an unbounded `Vec`, but quickly changed our mind (see [here](https://github.com/virto-network/virto-node/issues/292#issuecomment-1712425655)), since clearly using a `DoubleStorageMap` is a better choice for this use case.